ios/Classes/SwiftFlutterBarcodeScannerPlugin.swift

```diff
@@ -111,13 +111,38 @@ public class SwiftFlutterBarcodeScannerPlugin: NSObject, FlutterPlugin, ScanBarc
                                 
                             }
                         } else {
-                            let alert = UIAlertController(title: "Action needed", message: "Please grant camera permission to use barcode scanner", preferredStyle: .alert)
+                            let langCode = Locale.current.languageCode;
+                            var message: String;
+                            var title: String;
+                            var grant: String;
+                            var cancel: String;
+                            switch(langCode) {
+                                case "de":
+                                    message = "Zum Scannen von Codes brauchen wir Zugriff auf die Kamera.";
+                                    title = "Kameraberechtigungen zulassen?";
+                                    grant = "Ja";
+                                    cancel = "Abbrechen";
+                                    break;
+                                case "en":
+                                    message = "We need to use the camera to scan codes.";
+                                    title = "Allow camera permissions?";
+                                    grant = "Yes";
+                                    cancel = "Cancel";
+                                    break;
+                                default:
+                                    message = "We need to use the camera to scan codes.";
+                                    title = "Allow camera permissions?";
+                                    grant = "Yes";
+                                    cancel = "Cancel";
+                                    break;
```

Comment:
You can combine cases that have the same body:
```suggestion
                                case "en":
                                default:
                                    message = "We need to use the camera to scan codes.";
                                    title = "Allow camera permissions?";
                                    grant = "Yes";
                                    cancel = "Cancel";
                                    break;
```

Reply:
I tired. It needed to have body. I will check `case "en", default:`

Reply:
Yes you are right, a `,` is needed: https://docs.swift.org/swift-book/LanguageGuide/ControlFlow.html

Reply:
I decided to use `fallthrough`. Read it in some documentation that it can be used with the `default` case.